src/Sulu/Bundle/SecurityBundle/Security/LogoutSuccessHandler.php

```diff
@@ -29,7 +29,7 @@ class LogoutSuccessHandler implements LogoutSuccessHandlerInterface
 
     public function __construct(RouterInterface $router)
     {
-        @\trigger_error(__CLASS__ . '() is deprecated since version sulu/sulu 2.5 and will be removed in 3.0. Use LogoutEventSubscriber instead.', \E_USER_DEPRECATED);
+        @\trigger_deprecation('sulu/sulu', '2.5', __CLASS__ . '() is deprecated and will be removed in 3.0. Use LogoutEventSubscriber instead.');
```

Comment:
```suggestion
        @trigger_deprecation('sulu/sulu', '2.5', __CLASS__ . '() is deprecated and will be removed in 3.0. Use LogoutEventSubscriber instead.');
```

Reply:
Why not with the backslash? In #7030 I did every call with it and to my knowledge this makes it slightly faster because PHP checks the root namespace for the function immediately and not the local namespace first. But it works either way, so not a real problem. Just a bit of an inconsistency.

Reply:
The current `php-cs-fixer` config does only prefix `@internal` php methods: https://github.com/sulu/sulu/blob/ebaab29cb35155e44005305f3973764562da6eac/.php-cs-fixer.dist.php#L27. Didn't want to change this.